common/script/cron.js

```diff
@@ -41,7 +41,7 @@ function sanitizeOptions (o) {
     timezoneOffset = timezoneOffsetDefault;
   }
 
-  let now = o.now ? moment(o.now).zone(timezoneOffset) : moment().zone(timezoneOffset);
+  let now = o.now ? moment(o.now).utcOffset(timezoneOffset) : moment().utcOffset(timezoneOffset);
```

Comment:
The old and new functions never return the same value.

I have not tried to change it. I know how I want to do it but it should be done later. To much chance of chaos if it's bundled into the v3 changes. See https://github.com/HabitRPG/habitrpg/pull/7053#issuecomment-207950843